### ISSUE TITLE
Invalid format / value in next_config

### ISSUE THREAD

**Issue by thomasloubejac** [cmu-db/ottertune#185]
fab run_loops doesn't work for more that 1 iteration because next_config are generated with wrong values
```
$ pg_lsclusters
Error: invalid line 666 in /etc/postgresql/9.6/main/postgresql.conf: random_page_cost = 1.4103083961521482e+308
```
which is in the definition range so I guess it's wrong format.

after rectifying random_page_cost and seq_page_cost : 
```
$ tail /var/log/postgresql/postgresql-9.6-main.log 
2018-10-22 09:29:00.427 CEST [3207] LOG:  invalid value for parameter "max_wal_size": "17PB"
2018-10-22 09:29:00.427 CEST [3207] HINT:  Valid units for this parameter are "kB", "MB", "GB", and "TB".
2018-10-22 09:29:00.427 CEST [3207] LOG:  invalid value for parameter "min_wal_size": "21PB"
2018-10-22 09:29:00.427 CEST [3207] HINT:  Valid units for this parameter are "kB", "MB", "GB", and "TB".
2018-10-22 09:29:00.427 CEST [3207] FATAL:  configuration file "/etc/postgresql/9.6/main/postgresql.conf" contains errors
```

Knowing that I selected the t2.nano hardware on the website, this seems a little out of bound.
It also seems that changing session on the website in order to change hardware configuration doesn't change the way configurations are generated.

Is there a script to run to apply modifications for the next_config generation ?
something like 
I found `fab reset_website` and `python3 manage.py makemigrations sessions` but these don't seem to solve the problem.

**Comment by bohanjason** [comment]
Hi, It seems that the config is randomly generated. It will randomly choose value between min_val and max_val.  You may want to set the min_val and max_val in knobs fixture to the reasonable range.  It's too large for min_wal_size and max_wal_size according to your log.

**Comment by thomasloubejac** [comment]
There's a unit problem. shared_buffers is calculated in memory size but should be put in the next_conf without unit since it represents numbers of 8kB-sized blocks.
A second problem is that the memory related knobs are generated randomly between minval and maxval without checking if the total sum exceeds memory size.

I fixed the problems by adding a few lines in async_tasks.py and utils.py. I will make a pull request as soon as I figure out how to do it.

Finally postgres96 doesn't accept PB as valid units for *_wal_size so I had to reduce them in the knobs configuration file (/ottertune/server/website/website/fixtures/postgres96_knobs.json), this problem could be fixed by removing the PB unit in types.py but I guess this problem is specific to my config.
